mne/viz/epochs.py

```diff
@@ -561,8 +561,8 @@ def _prepare_mne_browse_epochs(params, projs, n_channels, n_epochs, scalings,
         types += [t] * len(inds[-1])
     pick_kwargs = dict(meg=False, ref_meg=False, exclude=[])
     if order is None:
-        order = ['eeg', 'eog', 'ecg', 'emg', 'ref_meg', 'stim', 'resp', 'misc',
-                 'chpi', 'syst', 'ias', 'exci']
+        order = ['eeg', 'seeg', 'eog', 'ecg', 'emg', 'ref_meg', 'stim',
+                 'resp', 'misc', 'chpi', 'syst', 'ias', 'exci']
```

Comment:
OK, ping me once this is merged. Thanks for the heads up

On Wed, Apr 20, 2016 at 12:52 PM, Eric Larson <notifications@github.com>
wrote:

> In mne/viz/epochs.py
> <https://github.com/mne-tools/mne-python/pull/3144#discussion_r60475829>:
>
> > @@ -561,8 +561,8 @@ def _prepare_mne_browse_epochs(params, projs, n_channels, n_epochs, scalings,
> >          types += [t] * len(inds[-1])
> >      pick_kwargs = dict(meg=False, ref_meg=False, exclude=[])
> >      if order is None:
> > -        order = ['eeg', 'eog', 'ecg', 'emg', 'ref_meg', 'stim', 'resp', 'misc',
> > -                 'chpi', 'syst', 'ias', 'exci']
> > +        order = ['eeg', 'seeg', 'eog', 'ecg', 'emg', 'ref_meg', 'stim',
> > +                 'resp', 'misc', 'chpi', 'syst', 'ias', 'exci']
>
> cc @choldgraf <https://github.com/choldgraf> you will want to work off of
> a fresh master after I merge this because there are more instances of seeg
> in here
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/mne-tools/mne-python/pull/3144/files/8b796aad31c74e1dcbf081dfa66a5379db468b51#r60475829>
>